apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -541,6 +545,41 @@ const BookingPage = ({
                   {eventType.description}
                 </p>
               )}
+              {eventType?.locations && (
+                <>
+                  {eventType.locations.length === 1 && (
+                    <p className="text-gray-600 dark:text-white">
+                      {Object.values(AppStoreLocationType).includes(
+                        eventType.locations[0].type as unknown as AppStoreLocationType
+                      ) ? (
+                        <Icon.FiVideo className="mr-[10px] ml-[2px] -mt-1 inline-block h-4 w-4 text-gray-500" />
+                      ) : (
+                        <Icon.FiMapPin className="mr-[10px] ml-[2px] -mt-1 inline-block h-4 w-4 text-gray-500" />
+                      )}
+                      {locationKeyToString(eventType.locations[0], t)}
+                    </p>
+                  )}
+                  {eventType.locations.length > 1 && (
+                    <div className="flex-warp flex text-gray-600 dark:text-white">
+                      <div className="mr-[10px] ml-[2px] -mt-1 ">
+                        <Icon.FiMapPin className="inline-block h-4 w-4 text-gray-500" />
+                      </div>
+                      <p>
+                        {eventType.locations.map((el: any, i: any, arr: any) => {
```

Comment:
```suggestion
                        {eventType.locations.map((el, i, arr) => {
```

These should be properly inferred